script/tool/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   `--packages` flag used by most other packages.
 - **Breaking change** Passing both `--run-on-changed-packages` and `--packages`
   is now an error; previously it the former would be ignored.
+- Formatting now skips files that contain a line that exactly
+  matches the string `// This file is hand-formatted.`.
```

Comment:
This got merged into the wrong section, it needs to be in a new one since 0.6.0 is published.

You should actually do a version bump in this PR since in order to use it in flutter/packages it'll need to be published (and then the flutter/packages pin rolled to pick it up)

Reply:
Oh, and it should be changed to "skips Dart files".

Reply:
Done.